resources/settings.xml

```diff
@@ -409,6 +409,11 @@
 						<heading>32001</heading>
 					</control>
 				</setting>
+                <setting id="omdb_apikey_access" type="boolean" label="32008" help="" parent="omdb_apikey">
```

Comment:
See comment above about localization. Same goes for the other settings with this label ID.